nailgun/entities.py

```diff
@@ -844,9 +844,11 @@ def __init__(self, server_config=None, **kwargs):
             'command': entity_fields.StringField(
                 required=True,
                 str_type='latin1',
+                default='top',
```

Comment:
This is an interesting addition. NailGun users won't encounter this unless they explicitly ask for fake data to be generated. And should Ontic development ever move forward, the fake data generation may be pulled out of NailGun entirely. Consequently, I'm willing to tailor the fake data generation in a way that suits Robottelo's needs.